client/my-sites/domains/domain-management/edit-contact-info-page/index.tsx

```diff
@@ -1,19 +1,176 @@
+		const previousPath = domainManagementEdit(
+			selectedSite?.slug,
+			selectedDomainName,
+			currentRoute
+		);
+
+		const items = [
+			{
+				label: translate( 'Domains' ),
+				href: domainManagementList( selectedSite?.slug, currentRoute ),
+			},
+			{
+				label: selectedDomainName,
+				href: previousPath,
+			},
+			{ label: translate( 'Edit contact infomation' ) },
+		];
+
+		const mobileItem = {
+			label: translate( 'Back' ),
+			href: previousPath,
+			showBackArrow: true,
+		};
+
+		return <Breadcrumbs items={ items } mobileItem={ mobileItem } />;
+	};
+
+	const renderContent = () => {
+		const domain = getSelectedDomain( { domains, selectedDomainName } );
+
+		if ( ! domain.currentUserCanManage ) {
+			return <NonOwnerCard domains={ domains } selectedDomainName={ selectedDomainName } />;
+		}
+
+		if ( domain.isPendingWhoisUpdate ) {
+			return <PendingWhoisUpdateCard />;
+		}
+
+		if ( domain.mustRemovePrivacyBeforeContactUpdate && domain.privateDomain ) {
+			return (
+				<EditContactInfoPrivacyEnabledCard
+					selectedDomainName={ selectedDomainName }
+					selectedSiteSlug={ selectedSite?.slug as string }
+				/>
+			);
+		}
+
+		return (
+			<EditContactInfoFormCard
+				domainRegistrationAgreementUrl={ domain.domainRegistrationAgreementUrl }
+				selectedDomain={ domain }
+				selectedSite={ selectedSite }
+				showContactInfoNote={ false }
+			/>
+		);
+	};
+
+	const renderSidebar = () => {
+		const supportLink = (
+			<ExternalLink
+				href={ localizeUrl(
+					'https://wordpress.com/support/domains/domain-registrations-and-privacy/#privacy-protection'
+				) }
+				target="_blank"
+				icon={ false }
+			/>
+		);
+		const icannLink = (
+			<ExternalLink
+				href="https://www.icann.org/resources/pages/contact-verification-2013-05-03-en"
+				target="_blank"
+				icon={ false }
+			/>
+		);
+		const explanationText1 = translate(
+			'{{icannLinkComponent}}ICANN{{/icannLinkComponent}} requires accurate contact information for registrants. This information will be validated after purchase. Failure to validate your contact information will result in domain suspension.',
+			{
+				components: {
+					icannLinkComponent: icannLink,
+				},
+			}
+		);
+		const explanationText2 = translate(
+			'Domain privacy service is included for free on applicable domains. {{supportLinkComponent}}Learn more{{/supportLinkComponent}}.',
+			{
+				components: {
+					supportLinkComponent: supportLink,
+				},
+			}
+		);
+		return (
+			<div className="edit-contact-info-page__sidebar">
+				{ /* eslint-disable-next-line wpcalypso/jsx-classname-namespace */ }
+				<div className="edit-contact-info-page__sidebar__title">
+					<p>
+						<strong>{ translate( 'Provide accurate contact information' ) }</strong>
+					</p>
+				</div>
+				{ /* eslint-disable-next-line wpcalypso/jsx-classname-namespace */ }
+				<div className="edit-contact-info-page__sidebar__content">
+					<p>{ explanationText1 }</p>
+					<p>{ explanationText2 }</p>
```

Comment:
Since these variables aren't used anywhere else, I think it would be better to define them directly in the block - also we would avoid the `1` and `2` endings.

Reply:
Yeah your right, my idea was to keep component code as "clean" as possibile, but in this case doesn't really make sense. I fixed it!